src/components/BreakoutRoomsEditor/SelectableParticipant.vue

```diff
@@ -4,35 +4,47 @@
 -->
 
 <template>
-	<div tabindex="0"
-		class="selectable-participant">
-		<input id="participant.attendeeId"
+	<label :for="participant.attendeeId"
+		class="selectable-participant"
+		:class="{ 'selectable-participant--selected': selected }">
+		<input :id="participant.attendeeId"
```

Comment:
It's a but dangerous to set `:id="participant.attendeeId"`, risky to duplicate IDs on the page.

But `for/id` are not needed here at all, because `input` is a descendant of the `label` here.